### ISSUE TITLE
Triton Scoreboard Problem  [3.2.2]

### ISSUE THREAD

**Issue by DarkyCat** [tritonmc/Triton#141]
**Describe the bug**
Scoreboard problem:

After the update 3.2.0 with compatibility with TAB, the ScoreBoard began to break for players, some of the information disappears or simply simply shifts. This is also not fixed on version 3.2.2.

Scoreboard problem screenshots:
- https://ibb.co/BT8tHFm
- https://ibb.co/tzRWRfx

**To Reproduce**
Update plugin to version 3.2.0-3.2.2

**Expected behavior**

**Screenshots (optional)**

**Server information (please complete the following information):**
 - Server Minecraft Version: Paper 1.16.5
 - Client Minecraft Version: 1.8 - 1.12.2 (On these versions, troubles.)
 - BungeeCord (specify if it's a fork) [yes or no] 
 - Triton Version: 3.2.2 (current using: 3.1.8)

**Additional context**
Add any other context about the problem here, including error logs.
If this bug is related to another plugin malfunction because of Triton, this is where you should indicate their versions.

**Comment by diogotcorreia** [comment]
Both screenshots don't load. Please try to upload them directly to GitHub.

Also, please indicate which scoreboard plugin you're using and, if possible, include a config that I can use to reproduce the issue. Otherwise, I can't fix your issue.

**Comment by DarkyCat** [comment]
Plugin: TAB
Version: 2.8.10
Author: NEZNAMY

![изображение](https://user-images.githubusercontent.com/50515714/107849295-2abb7500-6e1c-11eb-924d-fe52374e1f2c.png)
![изображение](https://user-images.githubusercontent.com/50515714/107849304-35760a00-6e1c-11eb-9d3b-67fc78e510ec.png)
These screenshots show that the information is skewed.

**Comment by diogotcorreia** [comment]
@DarkyCat Hi!
I'm sorry, I still can't reproduce this issue. Could you send me your scoreboard config for the TAB plugin? (most likely `premiumconfig.yml`)?
If you're uncomfortable sharing it here, DM it to me on Discord.

**Comment by DarkyCat** [comment]
[premiumconfig.zip](https://github.com/tritonmc/Triton/files/6004657/premiumconfig.zip)

**Comment by diogotcorreia** [comment]
Seems to be fine for me
![image](https://user-images.githubusercontent.com/7467891/108398897-6cc82a80-7211-11eb-81fe-cf02c8b619e8.png)

**Comment by DarkyCat** [comment]
If you roll back to version 3.1.8, the problem disappears.

**Comment by DarkyCat** [comment]
New error: https://mcpaste.io/cc2c2e3c89eee243

**Comment by DarkyCat** [comment]
![изображение](https://user-images.githubusercontent.com/50515714/108820568-2e9a8480-75de-11eb-8769-ca050c8bd4f6.png)

**Comment by DarkyCat** [comment]
If loaded without Triton, the ScoreBoard looks larger.
[14:54:00 INFO]: Plugins (48): AAC5, AbstractCooldown*, AdvancedEnchantments, BKCommonLib, Chatty, ClearLag, CMI, CMIEInjector*, CMILib, CMIPaperLib, CoreProtect, DailyRandomShop, DeluxeMenus, FarmLimiter, FastAsyncWorldEdit (WorldEdit), GSit, HeadDatabase, HolographicDisplays, HolographicExtension, JakesRTP, Keiko, LightCleaner, LiteBans, LuckPerms, MessageEditor, mMarketLib*, Multiverse-Core, OreRegenerator, Panilla, PlaceholderAPI, PluginManager, ProtocolLib, ProtocolSupport, RareAch*, SecuredWorlds, ServerNPC, ServerRestorer, SkinsRestorer, spark, Spartan, TAB, UClans, Vault, VoidGenerator, VoxelSniper, WGExtender, WorldGuard, WorldGuardExtraFlags

**Comment by DarkyCat** [comment]
with triton (KiteBoard another board plugin)
![изображение](https://user-images.githubusercontent.com/50515714/108824709-de262580-75e3-11eb-8e96-8e7fdcabd054.png)
without triton 
![изображение](https://user-images.githubusercontent.com/50515714/108824880-1594d200-75e4-11eb-88ce-f9002a848e10.png)

**Comment by DarkyCat** [comment]
I certainly know that since 1.13 the scoreboard limits have increased. But with Triton, he is trimmed.

**Comment by diogotcorreia** [comment]
I've finally been able to reproduce the issue. I was testing with ViaVersion instead of ProtocolSupport, which you only mentioned you were using a few messages ago.
It worked perfectly with ViaVersion, so I was struggling to reproduce it.

I've fixed two things:
- The toggle on the config for scoreboards wasn't really doing anything, I had forgotten to implement it.
- Since "scoreboard" translation is only enabled on Triton on 1.13 or above, I could remove the piece of code that was causing the issue since it was leftover from the older implementation that supported 1.8+.

This issue is now fixed and will be available in the next update.
